fancy-explosions/BangazonSite
# Js cleanup

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.



Fixes Issue # (link issue ticket here)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout js-cleanup
run it
go through app to check layout of headers spacing just all that



# Checklist:
